### PR TITLE
Deprecate local global keywords

### PR DESCRIPTION
**Pull request by ncphillips**
Implemeting [RFC 0006](https://github.com/tinacms/rfcs/pull/7)

### CODE REVIEW

File: packages/demo-gatsby/src/components/bio.js [hunk 1]
```diff
@@ -78,11 +78,13 @@ const Bio = () => {
     }
   `)
 
-  const [author] = useLocalJsonForm(data.dataJson, {
+  const [author, authorForm] = useJsonForm(data.dataJson, {
     label: "Author",
     fields,
   })
 
+  usePlugin(authorForm)
```
Comment (BPScott) [inline packages/demo-gatsby/src/components/bio.js:86]: This doesn't seem to be imported from anywhere? Should this be `useFormScreenPlugin`?

Comment (ncphillips) [reply packages/demo-gatsby/src/components/bio.js:86]: You're right on both counts! Thanks 😄

Comment (ncphillips) [reply packages/demo-gatsby/src/components/bio.js]: Thanks again, caught a couple other issues in the demos.

@all-contributors please add @BPScott for review

**Comment by ncphillips** [comment]
@all-contributors please add @BPScott for review

**Pull request merged**